CONTRIBUTING.md

```diff
@@ -117,7 +117,7 @@ Find more information in the [JabRef Wiki](https://github.com/JabRef/jabref/wiki
 Create a pull request on GitHub.
 For text inspirations, consider [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request).
 
-You can add the prefix `[WIP]` to indicate that the pull request is not yet complete, but you want to discuss something or inform about the current state of affairs.
+You can add the prefix `[WIP]` (which stands for Work in Progress) to indicate that the pull request is not yet complete, but you want to discuss something or inform about the current state of affairs.
```

Comment:
Please enquote "Work in Progress"

Reply:
This section is now obsolete/should be improved as github includes the option to mark a PR as draft (which should be the preferred option instead of adding [WIP]).

Reply:
Reading https://github.blog/2019-02-14-introducing-draft-pull-requests/, we cannot change the status of a PR to draft. So, I would prefer to keep `WIP`. Then, we can change in the middle of a conversation the PR to draft again.

(I like the new feature. Having no possibility to toggle the status after creation is a show stopper to me.)

Reply:
Mentioned draft PRs and retained WIP